### PR TITLE
feat: add ng cli version as build arg and add new tag

### PR DESCRIPTION
**Pull request by xchopin**
Add NG_CLI_VERSION as a build argument and create a new tag for 14.0.6

### CODE REVIEW

File: angular-cli/base/Dockerfile [hunk 1]
```diff
@@ -9,7 +9,7 @@ ARG CI_BUILD_ID
 ARG CI_BUILD_REF
 ARG CI_REGISTRY_IMAGE
 ARG CI_BUILD_TIME
-ARG NG_CLI_VERSION=latest
```
Comment (hoatle) [inline angular-cli/base/Dockerfile:12]: please don't change this, use it as default

**Review by hoatle — CHANGES_REQUESTED** [review]
@xchopin please check my comments, thank you.

File: angular-cli/base/Dockerfile (hunk 1 above)
Comment (xchopin) [reply angular-cli/base/Dockerfile]: ok fixed

**Comment by hoatle** [comment]
https://hub.docker.com/layers/angular-cli/teracy/angular-cli/14.0.6/images/sha256-2f003f2e00ac0bfd6614f1e9888c3dc4af58089fdabf6ec6668b20eac9424c92?context=explore is available, thank you @xchopin.

**Pull request merged**